src/frontend/Komponenter/Behandling/Brev/Brev.tsx

```diff
@@ -33,9 +35,10 @@ interface Props {
 const Brev: React.FC<Props> = ({ behandlingId }) => {
     const { axiosRequest, toast } = useApp();
     const [brevRessurs, settBrevRessurs] = useState<Ressurs<string>>(byggTomRessurs());
-    const { behandlingErRedigerbar, personopplysningerResponse, totrinnskontroll } =
+    const { behandling, behandlingErRedigerbar, personopplysningerResponse, totrinnskontroll } =
         useBehandling();
     const [kanSendesTilBeslutter, settKanSendesTilBeslutter] = useState<boolean>(false);
+    const [erKorrigeringUtenBrev, settErKorrigeringUtenBrev] = useState<boolean>();
```

Comment:
Må ærlig innrømme at hele denne sjekken i denne komponenten virker litt rotete. 
Hvor mye jobb er det å lage en egen "side" som heter `SendTilBeslutterUtenBrev` eller noe lignende?

Den skal jo isåfall bare inneholde `SendTilBeslutterFooter`, men kunne hatt en beskrivende tekst om at det ikke vil bli sendt ut brev for dette vedtaket? 

Nå ble det jo if-sjekk på alle metodene i denne komponenten 😬

Reply:
Fikset det sånn: https://github.com/navikt/familie-ef-sak-frontend/pull/1165/commits/f59b28ca2eb3303fcab8064e989d127339696384